### DISCUSSION TITLE
Show the path within the git repository

### DISCUSSION THREAD

**Discussion by NathanHuesken** [JanDeDobbeleer/oh-my-posh#5162] [Troubleshoot] [answered]
Hi,

Is it possible to show the path within the current git repsitory (when inside a git respository)?
So when I have a git repo at `~/projects/myproject` and I am at `~/porjects/myproject/foo/bar`, then only `foo/bar` is shown?

Thanks!

**Comment by JanDeDobbeleer** [comment, marked as answer]
```
"template": "{{ $segment := .Segments.Git }}{{ $icon := \"\uf1d2\" }}{{ if .Segments.Sapling.Dir }}{{ $segment = .Segments.Sapling }}{{ $icon = \"\ue22f\" }}{{ end }}{{ if $segment.Dir }} {{ $icon }} <i><b>{{ $segment.RepoName }}{{ if $segment.IsWorkTree}} \ue21c{{ end }}</b></i>{{if ne $segment.RepoName .Path}} \ueaf7{{ trimPrefix $segment.Dir .PWD }}{{ end }}{{ else }} \uea83 {{ path .Path .Location }}{{ end }}{{ if not .Writable }} \uf023{{ end }} "
```

this is my, very elaborate path template, which does what you want with `{{if ne $segment.RepoName .Path}} \ueaf7{{ trimPrefix $segment.Dir .PWD }}{{ end }}`

**Reply by RudolfVonKrugstein** [reply]
Nice! Thank you
